memento/cell_census_summary_cube.py

```diff
@@ -30,53 +30,49 @@
 # For testing
 # MIN_BATCH_SIZE = 1000
 
-CUBE_DIMS_OBS = [
+CUBE_TILEDB_DIMS_OBS = [
     "cell_type",
     "dataset_id",
+]
+
+CUBE_TILEDB_ATTRS_OBS = [
     "assay",
     "suspension_type",
     "donor_id",
     "disease",
     "sex"
 ]
```

Comment:
@mincheoly The above split of columns between "dims" and "attrs" allows us to select the columns that TileDB will index (in the sense of a relational database), allowing for efficient filtering on these columns. What is the access pattern for the cube? If there are 7 columns, will filtering always be performed on all columns? Or is it arbitrary?